rio_tiler/utils.py

```diff
@@ -298,6 +303,18 @@ def _tile_read(
     )
 
     vrt_transform, vrt_width, vrt_height = get_vrt_transform(src_dst, bounds)
+    out_window = None
+    if tile_edges_padding > 0:
+        vrt_transform = vrt_transform * Affine.translation(-tile_edges_padding, -tile_edges_padding)
+        orig__vrt_height = vrt_height
+        orig_vrt_width = vrt_width
+        vrt_height = vrt_height + 2 * tile_edges_padding
```

Comment:
I'm ok with having edges being a multiple of 2 but not sure if this really needed ? also if we keep it like that the doc string should mention it.

Reply:
I feel like the padding value makes most sense as a single edge so I'll update the docstring to make that clearer.